### DISCUSSION TITLE
how to add the "unlocked via SSH for headless" feature to an existing encrypted installation?

### DISCUSSION THREAD

**Discussion by tinker2much** [gitbls/sdm#173] [General]
v11.3: "Encrypted rootfs can now be configured and unlocked via SSH for headless systems"

Great! except I enabled encryption on an existing pi using the earlier version. Is there a path for me to get that functionality?

**Comment by gitbls** [comment]
Not currently. Reasonable question...I'll take a look at putting something together.

**Comment by gitbls** [comment]
Just closing the loop...tinker got this working on regular Debian with rootfs encryption enabled during Debian install. Woo hoo!
